.github/workflows/ci.yaml

```diff
@@ -406,3 +406,52 @@ jobs:
         with:
           name: release-images
           path: ./release-images
+
+  trivy-scan:
+    runs-on: ubuntu-20.04
+    needs: package-linux-amd64
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Download release package
+        uses: actions/download-artifact@v2
+        with:
+          name: release-packages
+          path: ./release-packages
+
+      - name: Build Docker image
+        run: ./ci/steps/build-docker-image.sh
```

Comment:
It might make more sense to fetch the docker image we build in `docker-amd64` instead of building them again in the Trivy workflow, I think

Reply:
Ooo didn't think of that. I'll see if I can do that!

Reply:
I'm going to get it working and then I'll refactor to fetch the docker image we build